src/api/helpers/get-total-contract-value-by-field/index.ts

```diff
@@ -0,0 +1,37 @@
+import { Context } from '../../types';
+
+const getTotalContractValueByField = async (context: Context, field: string, value: number): Promise<object | boolean> => {
```

Comment:
do we need some documentation at the top?

Reply:
Good spot - have moved this helper into a new directory and added documentation.